packages/firestore/src/core/sync_engine.ts

```diff
@@ -973,31 +940,28 @@ export class SyncEngine implements RemoteSyncer, SharedClientStateSyncer {
       switch (state) {
         case 'current':
         case 'not-current': {
-          return this.localStore.getNewDocumentChanges().then(
-            async changes => {
-              const synthesizedRemoteEvent = RemoteEvent.createSynthesizedRemoteEventForCurrentChange(
-                targetId,
-                state === 'current'
-              );
-              await this.emitNewSnapsAndNotifyLocalStore(
-                changes,
-                synthesizedRemoteEvent
+          try {
```

Comment:
You only need the extra braces on a `case` statement if there are declarations you want to keep local to the case. Any other construct that introduces a scope serves the same purpose, so the try/catch you've added now makes these braces redundant.

Reply:
I think technically they probably weren't needed before either...  so maybe prettier is just adding them automatically?

Reply:
yes, prettier is adding them.